src/frontend/src/components/WebformTable/common/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { OPTIONAL_HEADER_MARKER } from "src/components/DownloadMetadataTemplate/common/constants";
```

Comment:
This seems very similar to the code in `sec/components/DownloadMetadataTemplate`. Do you think we might want to keep the headers more/less in sync with that? I'm wondering if it might be better to import the base headers here and extend. That way if something in the base changes, we only have to update once.

Reply:
yeah.. i can do a sync between the 2 constants files.